doc/files/opensc.conf.5.xml.in

```diff
@@ -1902,6 +1914,14 @@ app <replaceable>application</replaceable> {
 						Extended example configuration file
 				</para></listitem>
 			</varlistentry>
+			<varlistentry>
+				<term>
+					<filename>@sysconfdir@/opensc/openssl.cnf</filename>
+				</term>
+				<listitem><para>
+						OpenSSL configuration used by OpenSC.
```

Comment:
Is it possible to add a reference to man 5 config, fips_config and x509v3_config for the definition of this file? Could also be relevant to the SEE ALSO section.

Reply:
Its a question that I am not sure if I can answer. The only use case we are aware so far is a FIPS, where we need to load some configuration to make openssl load FIPS provider. But it is true that it can be some shared configuration file among applications.

But in combination with, for example pkcs11-provider, the applications that will want to use pkcs11-provider will need a different configuration than OpenSC to make sure we do not recurse.

Using the opensc.conf to specify the path to the openssl.cnf sounds now better than using env variable (shared with the rest of the openssl applications) or specific path. Let me change that.

Reply:
Right, I forgot about the recursion problem with the configuration files. Since OPENSSL_CONF is already mentioned in `man 5 config`, we should *not* use this exact variable as it may point to the global OpenSSL configuration file with OpenSC as pkcs11 provider, which then leads to the recursion.

I would assume that the need for this kind of configuration is not very common so that an entry in the configuration file would be enough to cover this. No need to also check an environment variable.